parsec_cli/src/create_organization.rs

```diff
@@ -0,0 +1,72 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use clap::Args;
+use reqwest::Client;
+use serde_json::Value;
+use terminal_spinners::{SpinnerBuilder, DOTS};
+
+use libparsec::{BackendAddr, BackendOrganizationBootstrapAddr, OrganizationID};
+
+use crate::utils::*;
+
+#[derive(Args)]
+pub struct CreateOrganization {
+    /// OrganizationID
+    #[arg(short, long)]
+    organization_id: OrganizationID,
+    /// Backend address
+    #[arg(short, long)]
```

Comment:
There is no backend in what you mention.

Reply:
I don't want to add your suggestion because it is obviously the server addr.
I have specified in the comment which sort of address it is, i.e. it starts with `parsec://`

Reply:
I understand your point.

But we have a glossary to decide on which term to use when something is exposed to the client, as we will release the CLI to the client.

Reply:
> I understand your point.
> 
> But we have a glossary to decide on which term to use when something is exposed to the client, as we will release the CLI to the client.

Where did I use the `backend` word ?

Reply:
It seems that I've confused the `field name` with the `field types` :sweat: